tests/ociswrapper/ocis/ocis.go

```diff
@@ -271,3 +199,165 @@ func RunCommand(command string, inputs []string) (int, string) {
 
 	return c.ProcessState.ExitCode(), cmdOutput
 }
+
+func StartService(service string, envMap []string) {
+	// Initialize command args based on service presence
+	cmdArgs := []string{"server"} // Default command args
+
+	if service != "" {
+		cmdArgs = append([]string{service}, cmdArgs...)
+	}
+	// wait for the log scanner to finish
+	var wg sync.WaitGroup
+	wg.Add(2)
+
+	stopSignal = false
+	if retryCount == 0 {
+		defer common.Wg.Done()
+	}
+
+	cmd = exec.Command(config.Get("bin"), cmdArgs...)
+
+	if len(envMap) == 0 {
+		cmd.Env = append(os.Environ(), EnvConfigs...)
+	} else {
+		cmd.Env = append(os.Environ(), envMap...)
+	}
+	log.Println(fmt.Sprintf("%s", cmd.Env))
+
+	logs, err := cmd.StderrPipe()
+	if err != nil {
+		log.Panic(err)
```

Comment:
```suggestion
		log.Fatalf("some error message",err)
```
it is not recommended to use Panic as error as it crashes the code but fatal shut down code with proper error message. IMO, in go, other function would just collect and return error as one argument to main which handle that error. panic seems to forcely shut down system. I maybe wrong though

Reply:
:+1: we can refactor the use of Panic in the next PRs. These are existing codes so not related in this PR. Having separate PR would be good so that we can explain why no panic